client/controllers/feeds.coffee

```diff
@@ -13,7 +13,13 @@ Template.feeds.events
     if !/^(f|ht)tps?:\/\//i.test(feedUrl)
       feedUrl = "http://#{feedUrl}"
 
+    if Feeds.findOne(url: feedUrl)
```

Comment:
A check to see if the feed exists on another protocol would be a nice enhancement here. Http vs Https.